sky/skylet/providers/runpod/rp_helper.py

```diff
@@ -0,0 +1,116 @@
+'''
+RunPod library wrapper, formats the input/output of the RunPod library for SkyPilot.
+'''
+import os
+import json
+from typing import Dict
+
+import runpod
```

Comment:
To avoid causing import errors when a user do not have runpod installed, can we use the `sky.adaptors` design like the following?
https://github.com/skypilot-org/skypilot/blob/3213d3a14ecd86c3022f071e671fcf62c5ca48ba/sky/adaptors/gcp.py#L7-L26

```
from sky.adaptors import runpod

# whenever we need to use the `runpod` package, we call the following:
runpod.runpod()
```

In the adaptor file, we can do something like the following:
```python
runpod = None


def import_package(func):

    @functools.wraps(func)
    def wrapper(*args, **kwargs):
        global runpod
        if runpod is None:
            try:
                import runpod as _runpod
                runpod = _runpod
            except ImportError:
                raise ImportError('Failed to import dependencies for Runpod. '
                                  'Try: pip install "skypilot[runpod]"') from None
        return func(*args, **kwargs)

    return wrapper

@import_package
def runpod():
    return runpod
```